Resources/en.lproj/Localizable.strings

```diff
@@ -905,6 +905,12 @@
 "point_to_navigate" = "Navigate to";
 "start_of_track" = "Start of the track";
 "nearest_point" = "Nearest point";
+"plan_route_folder" = "Folder";
+"plan_route_change_folder" = "Change folder";
+"plan_route_select_folder" = "Select folder";
+"select_folder_descr" = "Select folder or add new one";
+"add_new_folder" = "Add new folder";
```

Comment:
Do we really need two strings? What is the difference between Add folder and Add new folder?